twine/settings.py

```diff
@@ -23,6 +25,16 @@
 from twine import utils
 
 
+def _setup_logging(verbose: bool) -> None:
+    """Initialize a logger based on verbosity available throughout twine."""
+    logger = logging.getLogger("twine")
```

Comment:
Following the suggestion in https://github.com/pypa/twine/pull/669#discussion_r446573415:

```suggestion
    root_logger = logging.getLogger("twine")
```

This will avoid future clashes if/when we add a global `logger` to this module.